elements/pfe-band/demo/index.html

```diff
@@ -261,21 +276,51 @@ <h3 slot="pfe-card--header">Aside: right body bottom</h3>
       </pfe-card>
     </pfe-band>
 
+ <pfe-band id="band" pfe-color="darker" pfe-aside-desktop="left" pfe-aside-height="full" pfe-img-src="https://www.redhat.com/profiles/rh/themes/redhatdotcom/img/header/header-hub-svsp-2000x1562.jpg">
```

Comment:
Should we reference a local background image instead to avoid a dependency on redhat.com?

Reply:
removed